api/src/test/java/org/openmrs/validator/AllergyValidatorTest.java

```diff
@@ -191,4 +191,13 @@ public void validate_shouldRejectNumericReactionValue() {
 		validator.validate(allergy, errors);
 		assertTrue(errors.hasErrors());
 	}
+	@Test
+	public void validate_shouldRejectNumericAllergenValue() {
+		Allergen allergen = new Allergen(AllergenType.DRUG, null , "45" );
+		allergy.setAllergen(allergen);
+		Errors errors = new BindException(allergy,"allergy");
+		validator.validate(allergy, errors);
+		assertTrue(errors.hasErrors());
+		assertTrue(errors.getAllErrors().stream()
+        .anyMatch(error -> "allergyapi.allergy.Allergen.cannotContainNumeric".equals(error.getCode())));	}
```

Comment:
Indent here should be with tabs.

Reply:
Also, the curly brace belongs on a new line. Please ensure you're following [the Java Conventions](https://wiki.openmrs.org/display/docs/Java+Conventions) and see [this to setup your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE) (basically, whatever mechanism will take an Eclipse formatting configuration).